scripts/backend_tests.py

```diff
@@ -238,8 +252,13 @@ def main():
     if parsed_args.test_target:
         all_test_targets = [parsed_args.test_target]
     else:
-        all_test_targets = _get_all_test_targets(
-            test_path=parsed_args.test_path)
+        if parsed_args.exclude_load_tests is not None:
```

Comment:
Hm, you don't need another branch. Just do:

     include_load_tests = (parsed_args.exclude_load_tests is None)